### PR TITLE
Document that addClass and removeClass change the attribute

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
Fixes gh-814

### CODE REVIEW

**Comment by AurelioDeRosa** [comment]
This PR is a bit redundant as it includes two paragraphs I've included in PR #813. However, I thought it was a good idea for two reasons. The first is that it reads better with a bit of context given by the copied paragraphs. The second is that it'll be easier to rebase when this branch will be merged into master.

**Comment by dmethvin** [comment]
For people who like to skim documents or quote them out of context it might be good to reword this to say, "Before jQuery version 3.0, ..." in that first sentence.

**Comment by AurelioDeRosa** [comment]
Wording updated.

**Comment by dmethvin** [comment]
LGTM! Don't know what's going on with the Travis run.

**Comment by AurelioDeRosa** [comment]
Cool. I'm merging it now.

About Trevis: this branch doesn't have a `.travis.yml` file.

**Comment by AurelioDeRosa** [comment]
Closed via 4651f4415ca00b8f9862d3a39a3eeea8600ddfff.
